src/Compiler/Driver/CompilerOptions.fs

```diff
@@ -697,6 +697,8 @@ let tagInt = "<n>"
 let tagPathMap = "<path=sourcePath;...>"
 let tagNone = ""
 let tagLangVersionValues = "{?|version|latest|preview}"
+let tagOptimizationDataValues = "{none|compress|file}"
```

Comment:
As mentioned above, `file` seems incorrectly named.  

Also today's `nooptimizationdata` is not quite `none` - it actually means `minimal` or `onlyessential`:
```
--nooptimizationdata                     Only include optimization information essential for implementing inlined
                                         constructs. Inhibits cross-module inlining but improves binary compatibility.
```

To summarize, for signature data, what you have today corresponds to this:
```
none = no signature data in resource
compress = compressed resource
file = uncompressed resource
```
and for optimization data, what you have today corresponds to this:
```
none = minimal optimization data in resource, and none at all if no signature data (but is it compressed???)
compress = compressed resource
file = uncompressed resource
```

So I think compression is actually orthogonal to selecting how much signature or optimization data is present.  It is likely better to have a single orthogonal switch `/compressmetadata`.